types/src/v17/network/mod.rs

```diff
@@ -65,7 +65,7 @@ pub struct GetConnectionCount(pub u64);
 pub struct GetNetTotals {
     /// Total bytes received.
     #[serde(rename = "totalbytesrecv")]
-    pub total_bytes_recieved: u64,
+    pub total_bytes_received: u64,
```

Comment:
While correct this is an API breaking change however the PR description claims its docs only. Can you pull this out into a separate PR please?